docs/newbs_getting_started.md

```diff
@@ -104,19 +104,13 @@ You can also try the `qmk-git` package from AUR:
 
 ###  ** FreeBSD **
 
-#### Prerequisites
-
-You will need to install Git and Python. It's possible that you already have both, but if not, run the following commands to install them:
-
-    pkg install git python3
-
-Make sure that `$HOME/.local/bin` is added to your `$PATH` so that locally installed Python packages are available.
-
 #### Installation
 
-Install the QMK CLI by running:
+Install the official FreeBSD package for QMK CLI by running:
 
-    python3 -m pip install --user qmk
+    pkg install -g "py*-qmk"
+    
+NOTE: remember to follow the instructions printed at the end of installation (use `pkg info -Dg "py*-qmk"` to show them again).
```

Comment:
The CLI is intended to be as easy to set up and use as possible, even on platforms where users generally know what they're doing (like FreeBSD and Linux). So if there is a way to automate any more of this, that would be helpful.

Reply:
There are only two instructions there:

1. Instructing the user to install the `python3` package so that there is a binary called `python3` on the system. The reason for this manual step is that the `python3` package cannot be dependent upon by a package. It is a package available strictly for user convenience.
2. Some instructions on how to set up rootless access to tty devices.

Only the python3 installation is necessary to really use the qmk_firmware repo.